ingenialink/eoe/network.py

```diff
@@ -128,7 +128,10 @@ def disconnect_from_slave(self, servo):
         if len(self.servos) == 0:
             self._stop_eoe_service()
             self._erase_config_eoe_service()
-            self._deinitialize_eoe_service()
+            try:
+                self._deinitialize_eoe_service()
```

Comment:
Inside the function, don't set `self._eoe_service_init = False` if the command fails

Reply:
Done!